tfutils/base.py

```diff
@@ -334,10 +336,6 @@ def run(sess,
         - thres_loss (float, default: 100)
             If loss exceeds this during training, HiLossError is thrown
     """
-    # initialize and/or restore variables for graph
```

Comment:
This is the main reason I didn't commit to master -- I see you put this code back into after I had originally removed it in the update I did a few weeks ago.   Do you think it's ok to just assume that initialization has already happened?  Since we're passing in a saver object anyhow, and since the Saver.__init__ will always cause initialization of variables (via load_model), I think this should be fine.  If you want to have something like this here, we might need to be careful to not quash any restored variables.